src/config/IPublicConfig.ts

```diff
@@ -51,6 +53,7 @@ export interface IPublicConfig {
   stylesheet?: IStyleSheetProps;
   cache?: boolean | ICacheProps;
   tsConfig?: string | IRawCompilerOptions;
+  customTransformers?: CustomTransformers;
```

Comment:
Because the privateConfig takes it's config values from the publicConfig and so it's propagated to the production pipeline: https://github.com/kyr0/fuse-box/commit/cfabfef38a0af9e2ea768e742ee75e7436f1f1b8#diff-68ccf37364cb8ccba6bdc3fa98aaba84R39

And for sure, to be able to set it from the outside:
https://github.com/kyr0/fuse-box/commit/cfabfef38a0af9e2ea768e742ee75e7436f1f1b8#diff-8ec425bf0945566e64ad90d79c388262R75